google/cloud/documentai_toolbox/wrappers/page.py

```diff
@@ -291,19 +291,19 @@ def _get_form_fields(
 
 
 def _table_rows_from_documentai_table_rows(
-    table_rows: List[documentai.Document.Page.Table.TableRow], text: str
-) -> List[str]:
+    table_rows: MutableSequence[documentai.Document.Page.Table.TableRow], text: str
```

Comment:
Why do you need a MutableSequence vs a List?

Reply:
mypy was saying that the types are incompatible. (Might be an implicit casting issue)

Reply:
MutableSequence is the type that `documentai.Document.Page.Table.table_rows` is from the Document AI client library